README-fr.md

```diff
@@ -16,9 +16,9 @@
 
 > Une expression régulière est un groupement de caractères ou symboles utilisés pour trouver un schéma spécifique dans un texte.
 
-Une expression régulière est un schéma qui est comparée à une chaîne de caractères de gauche à droite. Le mot "Expression régulière"
+Une expression régulière est un schéma qui est comparé à une chaîne de caractères de gauche à droite. Le mot "Expression régulière"
 est un terme entier, souvent abrégé par "regex" ou "regexp". Une expression régulière est utilisée pour remplacer un texte à l'intérieur
-d'une *string* (une chaîne de caractères), valider un formulaire, extraire une portion de string basée sur un schéma, et bien plus encore.
+d'une *string* (une chaîne de caractères), valider un formulaire, extraire une portion de chaine de caractères basée sur un schéma, et bien plus encore.
```

Comment:
It would be good write _chaine de caractères (string)_. Beyond its meaning, the most of the coders will understand the word _string_. 

So, just add the `(string)`

Reply:
It's done